include/uv.h

```diff
@@ -1144,6 +1144,17 @@ UV_EXTERN int uv_os_getenv(const char* name, char* buffer, size_t* size);
 UV_EXTERN int uv_os_setenv(const char* name, const char* value);
 UV_EXTERN int uv_os_unsetenv(const char* name);
 
+#ifdef MAXHOSTNAMELEN
+# define UV_MAXHOSTNAMESIZE MAXHOSTNAMELEN + 1
```

Comment:
Can you wrap this in parens? Without them e.g. `2 * UV_MAXHOSTNAMESIZE` doesn't produce the expected result (`2*255 + 1` instead of `2 * (255+1)`.)